authors/settings/base.py

```diff
@@ -138,6 +138,18 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        # 'authors.apps.authentication.backends.JWTAuthentication',
+        'authors.apps.authentication.backends.JWTAuthentication',
```

Comment:
Was this implemented @huxaiphaer , will not break the app? cc @reiosantos

Reply:
actually the app breaks , i have commented it out @inno-asiimwe , @reiosantos